### PR TITLE
LaTeX text formatter fixes

### PR DESCRIPTION
**Pull request by sciepsilon**
Some updates to the text format feature (PR #84, issue #81). 

- Use `<TranslatableText>` for all user-facing text
- Refactor into multiple files (this made using `<TranslatableText>` a lot easier)
- spacing and verbiage tweaks
- get rid of the Close button, since "Confirm" now works to update the tiers used after you make an initial selection
- rename button from "Format" to "LaTeX"

### CODE REVIEW

**Comment by sciepsilon** [comment]
latexbutton
tier selection
formatresult

**Comment by hollyyuqizheng** [comment]
I think this all looks great! Doing the refactoring for the TranslatableText definitely makes sense.

File: jsx/App/Stories/Story/Display/Latex/TierSelectionWindow.jsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    }); 
+
+  }
+
+  render() {
+    return (
+      <div className="tierSelectionSection">
+          <div className="tierSelectionWrapper">
+            <form className="tierSelectionForm" id={this.props.sentenceId}>
+              <p><TranslatableText dictionary={formatSelectTiersPromptText} /></p>
```
Comment (sciepsilon) [inline jsx/App/Stories/Story/Display/Latex/TierSelectionWindow.jsx:107]: rename `formatSelectTiersPromptText` -> `latexSelectTiersPromptText` for consistency

File: css/main.css [hunk 2]
```diff
@@ -199,20 +199,14 @@ td {
...
 .tierSelectionRow {
-	/* display: grid;
-	grid-template-columns: 1fr 7fr;
-	overflow-x: scroll; */
-	padding-bottom: 5%;
-	padding-top: 5%;
+	padding-bottom: 12px;
 }
 
 .formatResultSection {
```
Comment (sciepsilon) [inline css/main.css:212]: rename to `latexResultSection`?

File: jsx/App/Stories/Story/Display/LatexButton.jsx [hunk 3]
```diff
@@ -30,8 +30,8 @@ export class TextFormatButton extends React.Component {
     render() {
         return (
             <div class="textFormatSection">
```
Comment (sciepsilon) [inline jsx/App/Stories/Story/Display/LatexButton.jsx:32]: change `textFormatSection` name?

File: jsx/App/locale/LocaleConstants.jsx [hunk 4]
```diff
@@ -150,12 +150,42 @@ export const storySearchViewStoryText = {
   [FRANCAIS] : "Voir l'histoire",
 };
 
-// Texts for LaTeX conversion buttons
+// Texts for LaTeX conversion UI
 // Text on the format button
```
Comment (sciepsilon) [inline jsx/App/locale/LocaleConstants.jsx:154]: change comment to "Text on the LaTeX button" or better yet, delete it

**Review by sciepsilon — COMMENTED** [review]
A few places I missed in the "format" -> "latex" refactor

File: css/main.css [hunk 5]
```diff
@@ -177,49 +177,40 @@ td {
...
-	overflow-x: scroll; */
-	padding-bottom: 5%;
-	padding-top: 5%;
+	padding-bottom: 12px;
 }
 
-.formatResultSection {
-	background : rgb(215, 233, 252);
+.latexResultContainer {
 	overflow-x: scroll;
```
Comment (sciepsilon) [inline css/main.css:213]: Replace `scroll` -> `auto`. That way, the scroll bar only shows up when it's needed. (Too late to make this change now that we've already merged, but I'll keep it in mind for future PRs)

**Pull request merged**